fe1-web/jest/setupAfterEnv.js

```diff
@@ -0,0 +1,24 @@
+import FakeTimers from '@sinonjs/fake-timers';
+
+/**
+ * Sets up fake timers for tests.
+ *
+ * @remarks
+ * We use a library because there is an issue between jest.useFakeTimers and asynchronous tests.
+ * (see https://github.com/facebook/jest/issues/10221)
+ */
+
+const FIXED_SYSTEM_TIME = new Date(1620255600 * 1000); // 5 May 2021
+
+let clock;
+global.beforeAll(() => {
```

Comment:
Why not importing beforeAll from @jest/globals?

Reply:
It doesn't seem to be possible in this file, and the `global` assures that the setup is run globally (so before all the tests that we have and after all of them). I found it [there](https://stackoverflow.com/questions/48722930/running-a-global-test-setup-before-each-test-in-jest) and on other websites too.

Reply:
Thank you, I learned something :-)